CHANGELOG.md

```diff
@@ -8,6 +8,18 @@ and this project adheres to
 
 ## Unreleased
 
+#### Breaking Changes
```

Comment:
I'm confused. Why is this going into the release branch?

Reply:
Because we may still backport important fixes to the branch and we will want to mark them in the changelog.

But I'm also ok with omitting it for now and adding the titles when necessary.

Reply:
But then wouldn't those fixes/additions go into the "0.23.0" section? There shouldn't be an "Unreleased" section at all for the release branch as it's only for master right?

Reply:
No, they will initially go to the "Unreleased" section which will then become "0.23.1" once a new patch release is done.

Reply:
Of course we can directly create the "0.23.1" section once we find out that there are patches that need backporting.

Reply:
Ok, makes sense. I dropped this part, then.